wazo_sysconfd/plugins/networking_info/services.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import os
+
+
+def get_system_network_interfaces():
+    interfaces = os.listdir('/sys/class/net/')
```

Comment:
Not really. The user will know what interfaces to choose.
I'm not sure it would be a good idea to consider the current state of the interfaces when validating, as the user can configure an interface that will be activated later.